assets/js/helpers.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Change the URL parameters in the browser address bar on the fly
+ * Source: https://stackoverflow.com/a/10997390/403476
+ *
+ * @param {string} url      A valid URL, typically window.location.href
+ * @param {string} param    The name of the parameter to add/change
+ * @param {string} paramVal The value param should have
+ * @return {string}         url with updated parameters
+ */
+export function updateURLParameter(url, param, paramVal)
+{
+    let TheAnchor = null;
+    let newAdditionalURL = "";
+    let tempArray = url.split("?");
+    let baseURL = tempArray[0];
+    let additionalURL = tempArray[1];
+    let temp = "";
+    paramVal = typeof paramVal === 'undefined' ? '' : paramVal
+
+    if (additionalURL) {
+        let tmpAnchor = additionalURL.split("#");
+        let TheParams = tmpAnchor[0];
+            TheAnchor = tmpAnchor[1];
+        if(TheAnchor)
+            additionalURL = TheParams;
+
+        tempArray = additionalURL.split("&");
+
+        for (let i=0; i<tempArray.length; i++) {
+            if(tempArray[i].split('=')[0] != param) {
```

Comment:
Single quote here. Just noticed you're using double's everywhere else. Also, does that evaluation need to be lossy?